extension/js/background.js

```diff
@@ -1,6 +1,15 @@
 const RELAY_SITE_ORIGIN = "http://127.0.0.1:8000";
 
 browser.storage.local.set({ "maxNumAliases": 5 });
```

Comment:
Does this mean I could edit my local storage and give myself unlimited aliases? Or is that not really possible in the extension scope?

Reply:
The server side also checks how many aliases you already have before it makes a new one. So this is just for display purposes in the add-on.